src/Export/Scripts/bases.lua

```diff
@@ -132,6 +132,9 @@ directiveTable.base = function(state, args, out)
 			table.insert(implicitLines, line)
 			table.insert(implicitModTypes, modDesc.modTags)
 		end
+		if mod.Id == "SpearImplicitDisplaySpearThrow1" then
+			table.insert(implicitLines, "Grants Skill: Spear Throw")
```

Comment:
Interesting, so spear throw isn't an ItemInherentSkill like the others?

Reply:
For some reason, it is not. Maybe because it doesn't have more than 1 level?